sonar-ws/src/main/java/org/sonarqube/ws/client/issue/IssueFilterParameters.java

```diff
@@ -71,7 +71,12 @@
   public static final String FACET_MODE = "facetMode";
   public static final String FACET_MODE_COUNT = "count";
 
-  public static final String FACET_MODE_DEBT = "debt";
+  /**
+   * @deprecated since 5.5, replaced by {@link #FACET_MODE_EFFORT}
+   */
+  @Deprecated
+  public static final String DEPRECATED_FACET_MODE_DEBT = "debt";
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)